### DISCUSSION TITLE
Please add the volume slider to the TurboWarp player

### DISCUSSION THREAD

**Discussion by memesong** [ScratchAddons/ScratchAddons#5325] [Ideas]
I was just playing [ADOFAI Worlds 1-12](https://scratch.mit.edu/projects/728538791) and I realized while using the integrated SA Turbowarp player that there's no volume slider. Could you add the slider to the Turbowarp player or would that require reprogramming the entire player?

**Comment by GarboMuffin** [comment]
Follow https://github.com/TurboWarp/scratch-gui/issues/638
